### ISSUE TITLE
Add `Last Contributor`

### ISSUE THREAD

**Issue by lsaether** [w3f/polkadot-wiki-old#94]
Find some plug-in, or code it yourself, that pulls info from GitHub for the last account that committed that page and the timestamp of that commit. Display this information either above the heading of the page or directly below it like how the GH wiki does.

**Comment by laboon** [comment]
This plugin could be used for this: https://github.com/byrnereese/mkdocs-git-committers-plugin

However, it appears that this is called every time a page is loaded (you'd think it would just be added upon generation...) so it may cause some slowdowns in wiki loading time.

**Comment by lsaether** [comment]
Yeah that seems like it would be not good for people with slower connections. Ideally it would inject the information at build time and serve it statically, since the information (git commits) would only change at the same event which triggers builds (commit to master).
